queries.sql

```diff
@@ -0,0 +1,69 @@
+DROP TABLE IF EXISTS organization;
+DROP TABLE IF EXISTS channels;
+DROP TABLE IF EXISTS messages;
+
+CREATE TABLE organization (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    organization_name VARCHAR(128),
+
+);
+
+CREATE TABLE channels (
+     id INTEGER PRIMARY KEY AUTOINCREMENT,
+     channels_name VARCHAR(128) NOT NULL,
+ 
+);
+
+CREATE TABLE users (
+     id INTEGER PRIMARY KEY AUTOINCREMENT,
+     users_name VARCHAR(128) NOT NULL,
+);
+
+CREATE TABLE messages (
+    id INTEGER PRIMARY KEY AUTOINCREMENT, 
+    content VARCHAR(1024),
+    post_time DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    users_id INTEGER REFERENCES users(id),
+    channels_id INTEGER REFERENCES channels(id)
+
+);
+
+INSERT INTO organization (name) VALUES ('Lambda School');
+
+INSERT INTO users (name) VALUES ('Alice');
+INSERT INTO users (name) VALUES ('Bob');
```

Comment:
You try to insert into name and such column does not exist because You named it `users_name`.